tests/fixture/goal.ts

```diff
@@ -0,0 +1,14 @@
+export const GOAL_MOCK = {
+    id: 'MOCK_ID',
+    createdAt: 'MOCK_CREATED_AT',
+    title: 'MOCK_TITLE',
+    description: 'MOCK_DESCRIPTION',
+    createdBy: 'MOCK_CREATED_BY',
+    startsOn: 'MOCK_STARTS_ON',
+    endsOn: 'MOCK_ENDS_ON',
+    percentageCompleted: 0,
+    assignedBy: 'MOCK_ASSIGNED_BY',
+    status: 'MOCK_STATUS',
+    assignedTo: 'MOCK_ASSIGNED_TO',
+
+}
```

Comment:
The fixtures data should resemble the original data, rather than the current format.

Reply:
and also attach the issue ticket

Reply:
Sure